internal/pkg/consensus/expected.go

```diff
@@ -221,27 +221,18 @@ func (c *Expected) validateMining(ctx context.Context, st state.Tree, ts block.T
 		}
 
 		// Validate ElectionProof
-		numTickets := len(blk.Tickets)
-		result, err := c.IsElectionWinner(ctx, pwrTableView, blk.Tickets[numTickets-1], blk.ElectionProof, workerAddr, blk.Miner)
+		nullBlkCount := uint64(blk.Height) - prevHeight - 1
```

Comment:
I would go for `blk.Height.Uint64()` which will give us more control over who is doing arithmetic on it in case we ever change the size. Doing `uint64(x)` could drop bytes without us knowing.

Reply:
`blk.Height` is a `types.Uint64` which is [defined as a `uint64`](https://github.com/filecoin-project/go-filecoin/blob/master/internal/pkg/types/uint64.go#L32) under the hood.  I think casting to the underlying type is just a formality and won't ever lose us bytes.

Side note I think we're also safe from byte loss when the block's height field is decoded because we force decode into [an 8 byte array](https://github.com/filecoin-project/go-filecoin/blob/master/internal/pkg/types/uint64.go#L57)